src/core/js/views/adaptView.js

```diff
@@ -30,185 +30,165 @@ define([
       this.preRender();
       this.render();
       this.setupOnScreenHandler();
-    },
+    }
 
-    preRender: function() {},
+    preRender() {}
 
-    postRender: function() {
+    postRender() {
       this.addChildren();
-    },
+    }
 
-    render: function() {
-      Adapt.trigger(this.constructor.type + 'View:preRender', this);
+    render() {
+      const type = this.constructor.type;
+      Adapt.trigger(`${type}View:preRender`, this);
 
-      var data = this.model.toJSON();
+      const data = this.model.toJSON();
       data.view = this;
-      var template = Handlebars.templates[this.constructor.template];
+      const template = Handlebars.templates[this.constructor.template];
       this.$el.html(template(data));
 
-      Adapt.trigger(this.constructor.type + 'View:render', this);
+      Adapt.trigger(`${type}View:render`, this);
 
-      _.defer(function() {
+      _.defer(() => {
         // don't call postRender after remove
         if (this._isRemoved) return;
 
         this.postRender();
-        Adapt.trigger(this.constructor.type + 'View:postRender', this);
-      }.bind(this));
+        Adapt.trigger(`${type}View:postRender`, this);
+      });
 
       return this;
-    },
+    }
 
-    setupOnScreenHandler: function() {
-      var onscreen = this.model.get('_onScreen');
+    setupOnScreenHandler() {
+      const onscreen = this.model.get('_onScreen');
 
       if (!onscreen || !onscreen._isEnabled) return;
 
       this.$el
         .addClass('has-animation')
-        .addClass(onscreen._classes + '-before');
+        .addClass(`${onscreen._classes}-before`);
```

Comment:
Duplicate call to `addClass()` – can space-separate here:
```suggestion
      this.$el.addClass(`has-animation ${onscreen._classes}-before`);
```